modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -79,23 +79,31 @@ const SidebarPanelInfoView = ({
 
 	return (
 		<>
-			<Sidebar.Header subtitle={subType} title={title}>
-				{versions.map((version) => (
-					<div key={version}>
-						<ClayLabel displayType="info">
-							{`${Liferay.Language.get('version')} ${
-								version.version
-							}`}
-						</ClayLabel>
+			<Sidebar.Header title={Liferay.Language.get('content-info')} />
 
-						<ClayLabel displayType={version.statusStyle}>
-							{version.statusLabel}
-						</ClayLabel>
+			<Sidebar.Body>
+				<div className="mb-4 sidebar-dl sidebar-section">
```

Comment:
We can now use [Clay-space-utility-classes](https://clayui.com/docs/css/utilities/c-spacing.html#clay-sidebar) `.c-mb-4` they are the same as the Bootstrap ones but ~with 50% of sugar less~ without the `!important` attribute (I think there are more classes with `.m{direction}-{value}` or `.p{direction}-{value}` that could be changed)

Reply:
I can do it, anyway I'm wondering what if I want to overwrite some class spacing with `c-mb-2` for example. Boostrap's `!important` does the trick there, `c-mb-2` doesn't.

Reply:
If necessary of course we can use Bootstrap utility classes, it should be decided case by case https://github.com/liferay/liferay-frontend-guidelines/blob/master/css/when_to_use_css_utility_classes.md#2-the-important-attribute